core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -166,19 +165,12 @@ default long totalDurationNanos() {
     /**
      * Returns the preview of response content of the {@link Response}.
      * Note that the content preview needs to be enabled when configuring a {@link Server} or a {@link Client}
-     * to use this functionality.
+     * by {@link ContentPreviewingService} or {@link ContentPreviewingClient} decorators respectively.
```

Comment:
How about: `Note that a {@link Service} or a {@link Client} must be decorated with ... to enable the content preview.`